packages/permissions/src/hooks/use-is-authorized/use-is-authorized.ts

```diff
@@ -92,7 +91,8 @@ const useIsAuthorized = ({
         )
       );
     }
-    hasDemandeDataFences = hasAppliedDataFence({
+    hasDemandeDataFences = hasSomeDataFence({
+      actualPermissions,
```

Comment:
We need this now to match up against the acutal permission using the demanded data fences.